<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
    <line>
      <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
    </line>
  </reporter-output>
  <suite started-at="2024-04-16T09:51:09Z" name="Default suite" finished-at="2024-04-16T09:51:30Z" duration-ms="20305">
    <groups>
    </groups>
    <test started-at="2024-04-16T09:51:09Z" name="Default test" finished-at="2024-04-16T09:51:30Z" duration-ms="20305">
      <class name="reasearch.Test2">
        <test-method is-config="true" signature="setup()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:09Z" name="setup" finished-at="2024-04-16T15:21:09Z" duration-ms="14" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:09Z" name="method1" finished-at="2024-04-16T15:21:11Z" duration-ms="2066" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:11Z" name="method1" finished-at="2024-04-16T15:21:13Z" duration-ms="2017" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:13Z" name="method1" finished-at="2024-04-16T15:21:15Z" duration-ms="2011" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:15Z" name="method1" finished-at="2024-04-16T15:21:17Z" duration-ms="2016" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:17Z" name="method1" finished-at="2024-04-16T15:21:19Z" duration-ms="2006" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:19Z" name="method1" finished-at="2024-04-16T15:21:21Z" duration-ms="2004" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:21Z" name="method1" finished-at="2024-04-16T15:21:23Z" duration-ms="2015" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:23Z" name="method1" finished-at="2024-04-16T15:21:25Z" duration-ms="2008" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:25Z" name="method1" finished-at="2024-04-16T15:21:28Z" duration-ms="2016" status="SUCCESS_PERCENTAGE_FAILURE">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method signature="method1()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:28Z" name="method1" finished-at="2024-04-16T15:21:30Z" duration-ms="2019" status="FAIL">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:289)
	at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:246)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:667)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
            <line>
              <![CDATA[<a href = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png' target='blank'> I am from method</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<img height =200 weight =200 src = 'C:/Users/vdoppalapudi/Desktop/KT/Automation/TestNGrepo/TestBench/screen.png'>]]>
            </line>
          </reporter-output>
        </test-method> <!-- method1 -->
        <test-method is-config="true" signature="cleanup()[pri:0, instance:reasearch.Test2@5b7ea70d]" started-at="2024-04-16T15:21:30Z" name="cleanup" finished-at="2024-04-16T15:21:30Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
      </class> <!-- reasearch.Test2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
